front/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
This is the default readme for nestjs I think it does not add big value to our project and maybe we should replace it with a shorter readme explaining what the project is about